### PR TITLE
2020 

### PR DESCRIPTION
**Pull request by renefritze**
Structurally the site is done now. What's still missing:
 - content (check)
 - link check (I'll add doing this automatically in CI)
 - link 2019 index in 2020 page?

Have a look @pymor/pymor-devs, with `make serve` (uses podman if available, else docker)

### CODE REVIEW

File: _2020_pages/agenda.html [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+---
+    <!--background color-->
+    <script type="text/javascript">
+        document.write ('<body style="background: Gainsboro; background-attachment: fixed;">')
+    </script>
+
+    <div class="event black-frame">
+        <h1>{{ page.title }}</h1>
+        <p>
+            The pyMOR School will take place from Monday September 28th at 14:00 to Friday Ocotber 2nd at 12:00.
```
Comment (pmli) [inline _2020_pages/agenda.html]: typo
```suggestion
            The pyMOR School will take place from Monday September 28th at 14:00 to Friday October 2nd at 12:00.
```

File: _includes/2020_data/intro.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-12 text-center">
+                    <h2 class="section-heading">pyMOR School 2020</h2>
+
+                    <p>
+                        After a very succesful first iteration in Magdeburg in
+                        September 2019,
+                        the second annual pyMOR School will be held at the
+                        Münster Mathematics Cluster of Excellence from
```
Comment (pmli) [inline _includes/2020_data/intro.html]:
```suggestion
                        Mathematics Münster Cluster of Excellence from
```

**Review by pmli — COMMENTED** [review]
I like the new picture and the purple color. On the other hand, I'm conflicted about the blue color. It's nice, but it looks similar to the typical hyperlink color...

**Comment by pmli** [comment]
When I run `sudo make serve`, I get

```
docker run --rm --label=jekyll --volume /home/mp/Documents/mpi/misc/pymor/school.pymor.org:/srv/jekyll -v /home/mp/Documents/mpi/misc/pymor/school.pymor.org/.cache:/usr/local/bundle -it -p 127.0.0.1:4000:4000 jekyll/builder:stable bundle exec jekyll build
Traceback (most recent call last):
	11: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	10: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	 9: from /usr/local/lib/ruby/gems/2.6.0/gems/bundler-2.0.1/lib/bundler/setup.rb:20:in `<top (required)>'
	 8: from /usr/local/lib/ruby/gems/2.6.0/gems/bundler-2.0.1/lib/bundler.rb:107:in `setup'
	 7: from /usr/local/lib/ruby/gems/2.6.0/gems/bundler-2.0.1/lib/bundler/runtime.rb:20:in `setup'
	 6: from /usr/local/lib/ruby/gems/2.6.0/gems/bundler-2.0.1/lib/bundler/runtime.rb:108:in `block in definition_method'
	 5: from /usr/local/lib/ruby/gems/2.6.0/gems/bundler-2.0.1/lib/bundler/definition.rb:226:in `requested_specs'
	 4: from /usr/local/lib/ruby/gems/2.6.0/gems/bundler-2.0.1/lib/bundler/definition.rb:237:in `specs_for'
	 3: from /usr/local/lib/ruby/gems/2.6.0/gems/bundler-2.0.1/lib/bundler/definition.rb:170:in `specs'
	 2: from /usr/local/lib/ruby/gems/2.6.0/gems/bundler-2.0.1/lib/bundler/spec_set.rb:81:in `materialize'
	 1: from /usr/local/lib/ruby/gems/2.6.0/gems/bundler-2.0.1/lib/bundler/spec_set.rb:81:in `map!'
/usr/local/lib/ruby/gems/2.6.0/gems/bundler-2.0.1/lib/bundler/spec_set.rb:87:in `block in materialize': Could not find public_suffix-4.0.3 in any of the sources (Bundler::GemNotFound)
make: *** [Makefile:10: build] Error 1
```

**Comment by renefritze** [comment]
@pmli https://github.com/pymor/school.pymor.org/pull/6/commits/2058d22004ebec815858655c49b7f8a5a5c302c3 should fix that, sorry.

**Comment by pmli** [comment]
Now I get "The connection was reset" from Firefox when I open `localhost:4000`...

File: _2020_pages/register.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    <!--background color-->
+    <script type="text/javascript">
+        document.write ('<body style="background: Gainsboro; background-attachment: fixed;">')
+    </script>
+
+    <div class="event black-frame">
+        <h1>{{ page.title }}</h1>
+        <h2>Application</h2>
+
+        <p>The deadline is Friday <strong>August 29th 2020</strong> (AoE).</p>
```
Comment (pmli) [inline _2020_pages/register.html]: August 29 is Saturday.
What is "AoE"?

File: _data/twenty_20/faqs.yml [hunk 4]
```diff
@@ -0,0 +1,22 @@
+- question: "How much does participation cost?"
+  answer: "There is no participation fee. Lunches and coffee breaks will be
+  covered."
```
Comment (pmli) [inline _data/twenty_20/faqs.yml]: Is this true or just copied from 2019?

File: _data/twenty_20/faqs.yml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+- question: "Is there anything else for free?"
+  answer: "We can offer a limited number of rooms during the school (four
+  nights). For more details, see the <a href=\"../register\">registration
+  page</a> (<em>the registration deadline was August 31</em>)."
```
Comment (pmli) [inline _data/twenty_20/faqs.yml]: Same as above.

File: _data/twenty_20/faqs.yml [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+- question: "How much do I need to know about model reduction?"
+  answer: "Nothing. Basic knowledge of linear algebra and differential equations
+  is expected."
```
Comment (pmli) [inline _data/twenty_20/faqs.yml:6]: Did we consider increasing the requirements? Or just doing shorter lectures?

File: _2020_pages/agenda.html [hunk 7]
```diff
@@ -0,0 +1,161 @@
...
+            {% for agenda in site.data.twenty_20.agendas_5 %}
+            <tr>
+                <td>{{ agenda.time }}</td>
+                <td>{{ agenda.info }}</td>
+            </tr>
+            {% endfor %}
+        </table> -->
+
+        <h2>School materials</h2>
+        <ul>
```
Comment (pmli) [inline _2020_pages/agenda.html:113]: Maybe also comment out this part? Or add a remark that school materials will be added here?

File: _2020_pages/venue.html [hunk 8]
```diff
@@ -0,0 +1,134 @@
...
+          </address>
+      </p>
+
+      <h2>Directions</h2>
+      <h3>By plane</h3>
+        <p>The <a href="https://www.dus.com/en" class="ext" target="_blank">Airport Düsseldorf </a>is  the nearest
+          airport equipped with an  international transport network.  You can reach Münster from airport by  train in
+           about 90 minutes. For  more details see the Travel Service of
+           <a href="https://www.bahn.com/en/view/index.shtml" class="int">"Deutsche Bahn"</a>. Alternatively,
+           the Airport Münster-Osnabrück, which has  a significant smaller transport system, is located about 30 km
```
Comment (pmli) [inline _2020_pages/venue.html]:
```suggestion
           the <a href="https://www.fmo.de/en/" class="ext" target="_blank">Airport Münster-Osnabrück</a>, which has  a significantly smaller transport system, is located about 30 km
```

File: _2020_pages/venue.html [hunk 9]
```diff
@@ -0,0 +1,134 @@
...
+          Alternatively, you can also travel via Dortmund and change trains there. In this case you leave from platform
+          FERN 6 and need about 2.5 hours for the whole journey.</li>
+        </ul>
+
+      <h3>By public transport</h3>
+        <p>Münster is well connected by train and bus. See the website of
+            <a href="https://www.bahn.com/en/view/index.shtml" class="ext" target="_blank">"Deutsche Bahn" </a>
+            for more travel details.
+        </p>
+        <h4>From the train station to Seminarraumzentrum</h4>
```
Comment (pmli) [inline _2020_pages/venue.html]: For some reason, `h4` is blue, but `h3` is black. Is this on purpose?

File: _2020_pages/venue.html [hunk 10]
```diff
@@ -0,0 +1,134 @@
...
+                      </li>
+                      <li>
+                          <a href="https://goo.gl/maps/28f7th3UMDR2">
+                              Google maps route
+                          </a>
+                      </li>
+                  </ul>
+              </li> -->
+          </ul>
+      </p> -->
```
Comment (pmli) [inline _2020_pages/venue.html]: This `-->` is visible.

**Review by pmli — COMMENTED** [review]
Great, it's working now. I wrote some comments below.

BTW, any idea why the text on the agenda and venue pages is black instead of blue like in the about page?

File: _2020_pages/register.html (hunk 3 above)
Comment (renefritze) [reply _2020_pages/register.html]: Yeah, that should be the Friday then. AoE = Anywhere on earth, or UTC-12

File: _data/twenty_20/faqs.yml (hunk 4 above)
Comment (renefritze) [reply _data/twenty_20/faqs.yml]: Both true and copied :wink:

File: _data/twenty_20/faqs.yml (hunk 5 above)
Comment (renefritze) [reply _data/twenty_20/faqs.yml]: This is actually just copied. Good catch :+1:

File: _2020_pages/agenda.html (hunk 7 above)
Comment (renefritze) [reply _2020_pages/agenda.html:113]: Right.

File: _2020_pages/venue.html (hunk 9 above)
Comment (renefritze) [reply _2020_pages/venue.html]: Nope

File: _2020_pages/venue.html (hunk 8 above)
Comment (renefritze) [reply _2020_pages/venue.html]: :+1:

File: _data/twenty_20/faqs.yml (hunk 6 above)
Comment (renefritze) [reply _data/twenty_20/faqs.yml:6]: Just shorter lectures I think

File: _2020_pages/register.html (hunk 3 above)
Comment (pmli) [reply _2020_pages/register.html]: Ok. When I google "AoE", I don't get that (mostly it's about Age of Empires)...

File: _data/twenty_20/faqs.yml (hunk 4 above)
Comment (pmli) [reply _data/twenty_20/faqs.yml]: Ok, great :+1:

**Comment by renefritze** [comment]
Haven't looked into the color stuff you noticed yet, @pmli

**Comment by renefritze** [comment]
Should be good to go now from my end.

File: _data/twenty_19/faqs.yml [hunk 11]
```diff
@@ -1,11 +1,5 @@
 - question: "How much does participation cost?"
-  answer: "There is no participation fee. Lunches and coffee breaks will be
-  covered."
-
-- question: "Is there anything else for free?"
-  answer: "We can offer a limited number of rooms during the school (four
-  nights). For more details, see the <a href=\"../register\">registration
-  page</a> (<em>the registration deadline was August 31</em>)."
+  answer: "There is no participation fee."
```
Comment (pmli) [inline _data/twenty_19/faqs.yml]: @renefritze It looks like the wrong year was updated.

Comment (renefritze) [reply _data/twenty_19/faqs.yml]: D'oh. Fixed now!

File: _2020_pages/register.html [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+
+        <p>
+            To register, send an <strong>email</strong> to
+            <a href="mailto:school@pymor.org"><strong>school@pymor.org</strong></a>:
+        </p>
+        <ul>
+            <li>with an attached <strong>short motivation letter</strong>.
+        </ul>
+
+        <h2>Short motivation letter</h2>
```
Comment (pmli) [inline _2020_pages/register.html]: I guess a list with a single item and a separate section are not necessary.

File: _data/twenty_20/faqs.yml (hunk 6 above)
Comment (pmli) [reply _data/twenty_20/faqs.yml:6]: Ok, that's probably better.

**Comment by pmli** [comment]
Besides the comment above, it looks good to me.

**Comment by renefritze** [comment]
This is now going live. Thanks for review, @pmli !

**Pull request merged**